perception/autoware_traffic_light_arbiter/include/autoware/traffic_light_arbiter/traffic_light_arbiter.hpp

```diff
@@ -40,6 +41,12 @@ class TrafficLightArbiter : public rclcpp::Node
   using TrafficSignalArray = autoware_perception_msgs::msg::TrafficLightGroupArray;
   using TrafficSignal = autoware_perception_msgs::msg::TrafficLightGroup;
 
+  struct ExternalTrafficLightInfo
```

Comment:
How about using TrafficSignalArray msg?https://github.com/autowarefoundation/autoware_msgs/blob/f49c35c7a7b0110ad7bf5d91f67967833fd1c0d9/autoware_perception_msgs/msg/TrafficSignalArray.msg

Reply:
@shmpwk thank you, good point. I will make the update. Note that it is TrafficLightGroupArray, not TrafficSignalArray, because `  using TrafficSignalArray = autoware_perception_msgs::msg::TrafficLightGroupArray;
`